templates/hoover-deps.nomad

```diff
@@ -55,7 +56,7 @@ job "hoover-deps" {
 
   group "es-master" {
     ${ continuous_reschedule() }
-    ${ group_disk() }
+    ${ group_disk(200) }
```

Comment:
why'd you need this?

Reply:
I had an error related to disk space which I thought was resolved by this. Something like log size exceeded or whatever. But apparently it wasn't related and resolved by something else. Removed it now.